src/main/scala/exercises/06_Collections.scala

```diff
@@ -139,23 +145,31 @@ object Collections {
    * As usual, various ways exist: pattern matching, folding, ...
    */
   def maxElementInList(l: List[Int]): Int = {
-    error("fix me")
+    l.foldLeft(l.head)((acc: Int, max: Int) => if (acc > max) acc else max)
   }
 
   /**
    * Calculate the sum of the equally position elements
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
```

Comment:
This would read more clearly using case statements and pattern matching:

```scala
case Nil l2 => l2
case l1 Nil => l1
case (x::xs) (y::ys) => (x + y)::sumOfTwo(xs, ys)
```